### ISSUE TITLE
Longitudinal shift in diabetic wound microbiota correlates with prolonged skin defense response.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#18]
Longitudinal shift in diabetic wound microbiota correlates with prolonged skin defense response.
Grice EA, Snitkin ES, Yockey LJ, Bermudez DM; NISC Comparative Sequencing Program, Liechty KW, Segre JA.
Proc Natl Acad Sci U S A. 2010 Aug 17;107(33):14799-804. doi: 10.1073/pnas.1004204107. Epub 2010 Jul 28. Erratum in: Proc Natl Acad Sci U S A. 2010 Oct 12;107(41):17851.
http://www.ncbi.nlm.nih.gov/pubmed/20668241

**Comment by nbaxter13** [comment]
In this paper from July Segre’s lab, they investigated changes in the bacterial community and host gene expression in a mouse model of diabetic wound healing.  The made wounds in the skin of db/db mice (Leptin receptor deficient mice, with wound healing defect similar to diabetics) and db/+ mice (heterozygous controls), and they monitored the wound healing process and shifts in the bacterial community.  At baseline the two genotypes of mice harbored different bacterial communities.  During the wound healing process the db/+ mice underwent a shift in the skin microbiome at the wound site, but the community recovered to baseline as the skin fully healed around 2 weeks post wounding.  In contrast, the skin microbiome of db/db didn’t shift quite as much in response to the wound, which was only ~60% healed at 2 weeks and fully healed at 4 weeks.  However, the microbiome of the db/db mice never fully returned to baseline after the skin healed.  Analysis of host gene expression showed that both types of mice had a spike in expression of genes associated with innate immunity and wound healing in response to the wound.  However, db/db mice showed a prolonged upregulation of these genes (Maybe because the wound was still healing?).  Finally they found a correlation between Staphylocci and innate defense genes.

The authors argue that the dysbiotic microbiota of the db/db mice is what’s responsible for the prolonged wound healing.  I may have missed something, but I really don’t see how they can make that claim.  Couldn’t the prolonged dysbiosis and prolonged expression of wound healing genes just as easily be a result of, I don’t know…  prolonged wound healing!?  In the discussion, they talk about Staphylocci being **associated** with slow wound healing and that the skin microbiome can modulate host immunity.  Personally, I think their hypothesis will turn out to be correct.  I just don’t think they’ve done anything to shore it up in this study.  I know I’m being hypocritical for complaining about relying solely on correlation, but couldn’t they at least do a cohousing experiment to show that the phenotype is transferrable?

tl;dr - Slow-healing wounds of diabetics have a dysbiotic microbiome and prolonged expression of wound healing and innate defense genes.

Overall: 6
Importance: 6
Innovation: 5
Who should read:
Tags: skin microbiome, innate immunity, diabetes

**Comment by pschloss** [comment]
Grice and colleagues in the Segre lab used a mouse model of diabetes to track the progression of the skin microbial community before, during, and after excising a patch of skin from the back of the mice. As expected, the wound in the diabetic mice actually gets larger with time before starting to heal. In contrast, the control mice start to heal almost immediately. So why would they expect bacteria to be involved in this process? As they point out nonhealing diabetic wounds seem to be driven by chronic inflammation. Furthermore, necrotic tissue and microaerophilic conditions make for a nice environment in which bacteria could grow. Now five years old, the paper used Sanger sequencing of 16S rRNA genes and genus-level classification of their sequencing data. It shows a reduction and increase in diversity with wound healing. They also saw large shifts in Staphylococcus with time. They used host transciptomic profiling during the healing process and focused in on genes involved in the immune response, host defense, and  wound healing. They saw a lot of the things that you'd expect to see. I think it is easy to dismiss this paper as being overly descriptive and unfortunate that it was done a few years too early. But I think that kind of misses the overall point. Yes, the microbiome analysis was *very* descriptive and the transcriptomic work is kind of predictable. But to me, the exciting part of this paper was that they were doing multi-omics before it was a buzzword that no one knew what to do with. Here, they connected the two 'omics technologies to show a correlation between the relative abundance of Staphyloccus and host defense genes. That's pretty cool and suggests an avenue for future experiments.

One thing I wondered as I went through was perhaps it isn't necessarily the genotype, but rather the type of bacteria on the skin. Figure 1 makes it pretty clear that the mice start with very different skin communities. They also state "on average 40 times more bacteria populated db/db skin as compared with db/+ skin". One thing to look for in these types of experiments is whether the experimental group and control groups were derived from the same parents. To their credit, these were. As they state in the methods the db/db and db/+ mice were litter mates that were separated for the experiment. While it would have been great to keep a representative of each genotype together in the cage, I suspect this is impossible since they'd probably start gnawing at each other's wounds.

As I mentioned above, they have clearly shown a shift in the microbiome that tracks with changes in the host response. Returning to the original hypothesis, it would have been nice to see a few additional experiments. First and perhaps most obvious, it would have been nice to take a WT germ-free mouse and compare its rate of wound  healing to that of a conventionally raised mouse. Are the rates of healing slower or faster with bacteria? Their hypothesis would seem to indicate that the wrong type of bacteria hinder the recovery. Second, perhaps it's the db/db genotype as they also hypothesize. Repeat my experiment using db/db germ-free mice and compare the rates of healing to conventional mice.


Overall: 6
Importance: 6
Innovation: 7
Who should read: @mjenior @alyxwithay 
Tags: mouse models, microbiome, germ free, skin
